client/src/pages/Maintenance/Maintenance.scss

```diff
@@ -0,0 +1,85 @@
+@import '../../scssStyles/mixins.scss';
+
+.full-page {
+  width: 100000vh;
+  height: 100000vh;
```

Comment:
this part looks very weird but the gradient didn't cover the full screen when i used 100% or 100vh...

Reply:
No problem, weird CSS sometimes does the job